react/src/pages/analysis/Analyses.tsx

```diff
@@ -448,25 +623,52 @@ export default function Analyses() {
-                                <div style={{ marginLeft: '24px' }}>
-                                    <Chip label="Completed" clickable className={classes.chip} onClick={() => setChipFilter(PipelineStatus.COMPLETED)} />
-                                    <Chip label="Running" clickable className={classes.chip} onClick={() => setChipFilter(PipelineStatus.RUNNING)} />
-                                    <Chip label="Pending" clickable className={classes.chip} onClick={() => setChipFilter(PipelineStatus.PENDING)} />
-                                    <Chip label="Error" clickable className={classes.chip} onClick={() => setChipFilter(PipelineStatus.ERROR)} />
-                                    <Chip label="Cancelled" clickable className={classes.chip} onClick={() => setChipFilter(PipelineStatus.CANCELLED)} />
-                                    <IconButton onClick={() => setChipFilter("")}> <Cancel/> </IconButton>
+                                <div style={{ marginLeft: "24px" }}>
+                                    <Chip
+                                        label="Completed"
+                                        clickable
+                                        className={classes.chip}
+                                        onClick={() => setChipFilter(PipelineStatus.COMPLETED)}
+                                    />
+                                    <Chip
+                                        label="Running"
+                                        clickable
+                                        className={classes.chip}
+                                        onClick={() => setChipFilter(PipelineStatus.RUNNING)}
+                                    />
+                                    <Chip
+                                        label="Pending"
+                                        clickable
+                                        className={classes.chip}
+                                        onClick={() => setChipFilter(PipelineStatus.PENDING)}
+                                    />
+                                    <Chip
+                                        label="Error"
+                                        clickable
+                                        className={classes.chip}
+                                        onClick={() => setChipFilter(PipelineStatus.ERROR)}
+                                    />
+                                    <Chip
+                                        label="Cancelled"
+                                        clickable
+                                        className={classes.chip}
+                                        onClick={() => setChipFilter(PipelineStatus.CANCELLED)}
+                                    />
+                                    <IconButton onClick={() => setChipFilter("")}>
+                                        {" "}
+                                        <Cancel />{" "}
```

Comment:
Only weirdness I could find on the frontend. Not really an issue to address right now, but we should double-check our code after formatting on our own branches.

Reply:
It looks like this happened because there was a space before and after when this was one line, and sometimes that's significant when rendered. I'll get rid of it.